src/Perspex.Controls/Panel.cs

```diff
@@ -6,145 +6,177 @@
 using System.Collections.Specialized;
 using System.Linq;
 using Perspex.Collections;
+using Perspex.Media;
 
 namespace Perspex.Controls
 {
-    /// <summary>
```

Comment:
Can you fix your editor settings so that your changes don't replace the entire file contents?